packages/blocks/divider/webpack-dev-server.config.js

```diff
@@ -3,7 +3,7 @@ const webpack = require("webpack");
 const config = require("./webpack.config");
 
 module.exports = {
-  entry: "./src/webpack-dev-server.js",
+  entry: "./src/webpack-dev-server.tsx",
```

Comment:
It maybe worth making the same change in other blocks for consistency. Otherwise, the blocks diverge and become a bit harder to manage.

Reply:
Fair point. Some other blocks do have the `.tsx` extension already though, where type-checking was required. Captured here: https://app.asana.com/0/1201707629991380/1201820398013214/f